Tests/ScenarioTester/ScenarioTester/Request.cs

```diff
@@ -0,0 +1,187 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using static System.FormattableString;
+
+namespace Microsoft.TestCommon.Utilities.ScenarioTester
```

Comment:
Can we avoid putting the Microsoft name in the namespace?

Reply:
Fixed.